bower.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "adapt-contrib-matching",
-  "version": "2.1.1",
+  "version": "2.1.2",
```

Comment:
#99 already handles this (and version number in README too) so no need for you to do in this PR